<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcedureConfigurationWidget</class>
 <widget class="QWidget" name="ProcedureConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>735</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(208, 208, 208);
background-color: rgb(83, 83, 83);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="2,1">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2">
     <item>
      <widget class="QGroupBox" name="procedureListGroupBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Procedure List</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="procedureListTableView">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="AddProcedureBtn">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteBtn">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="procedureDetailsGroupBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Procedure Details</string>
       </property>
       <widget class="QLabel" name="procedureNameLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>41</y>
          <width>47</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>41</y>
          <width>150</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="codeLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>80</y>
          <width>42</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Code</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="codeLineEdit">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>80</y>
          <width>150</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="codeSchemaLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>119</y>
          <width>109</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Code Schema</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="codeSchemaLineEdit">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>119</y>
          <width>150</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="codeMeaningLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>158</y>
          <width>116</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Code Meaning</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="codeMeaningLineEdit">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>158</y>
          <width>150</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="bodyPartLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>197</y>
          <width>76</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Body Part</string>
        </property>
       </widget>
       <widget class="QComboBox" name="bodyPartComboBox">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>197</y>
          <width>150</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="anatomicRegionLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>236</y>
          <width>136</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Anatomic Region</string>
        </property>
       </widget>
       <widget class="QComboBox" name="anatomicRegionComboBox">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>236</y>
          <width>150</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="printOrientationLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>275</y>
          <width>130</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Print Orientation</string>
        </property>
       </widget>
       <widget class="QComboBox" name="printOrientationComboBox">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>275</y>
          <width>150</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Landscape</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Portrait</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="printFormatLbl">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>314</y>
          <width>98</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Print Format</string>
        </property>
       </widget>
       <widget class="QComboBox" name="printFormatComboBox">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>314</y>
          <width>154</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STANDARD\1,1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STANDARD\1,2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STANDARD\2,1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STANDARD\2,2</string>
         </property>
        </item>
       </widget>
       <widget class="QCheckBox" name="trueSizeCheckBox">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>353</y>
          <width>97</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>True Size</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="procedureViewGroupBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Procedure's View List</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="allViewGroupBox">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="title">
         <string>All views</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListView" name="allViewListView">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeViewBtn">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addViewBtn">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="selectedViewGroupBox">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Selected Views</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListView" name="selectedViewListView">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
